sklearn/utils/_weight_vector.pyx.tp

```diff
@@ -18,16 +18,24 @@ dtypes = [('64', 'double', 1e-9),
 
 }}
 
+# cython: language_level=3
```

Comment:
Hmm we do set language_level globally here:

https://github.com/scikit-learn/scikit-learn/blob/21eb4686c5a53401b10e815d2f08ef8f090283e1/sklearn/_build_utils/__init__.py#L79

but it is a little nicer to set it explicitly.

Reply:
As per the other thread, it helps generating code for this file solely if needed, but probably this should be removed.